django/db/models/fields/tuple_lookups.py

```diff
@@ -129,6 +130,20 @@ def as_sql(self, compiler, connection):
         return super().as_sql(compiler, connection)
 
 
+class TupleComparisonMixin:
+    def as_oracle(self, compiler, connection):
+        """
+        Regardless of whether a subquery has been limited to return
+        a single row, this error is raised for <, <=, >, and >=:
+        ORA-01796: this operator cannot be used with lists
+        """
+        if isinstance(self.rhs, Query) and self.rhs.subquery:
+            lookup = self.lookup_name
+            msg = f"{lookup} for composite fields is not supported on this backend"
+            raise NotSupportedError(msg)
+        return super().as_oracle(compiler, connection)
```

Comment:
I was surprised this event worked but it seems to delegate to `Lookup.as_oracle` which is unnecessary in this case but works.